python/Ganga/GPIDev/Lib/Config/Config.py

```diff
@@ -266,10 +266,14 @@ def config_file_as_text():
                                 def_value = "\n# ".join(def_lines)
                         except AttributeError as err:
                             pass
+                    print 'o: ', o, ' def_value: ', def_value, ' value: ', value
```

Comment:
Yes I'll get rid of that for now and put it in properly for copying over the gangarc.